index.js

```diff
@@ -1,4 +1,43 @@
-import Books from './Books.js';
+import Books from './modules/Books.js';
+import {
+  booksList,
+  addBooks,
+  contactUs,
+  listLink,
+  addNewLink,
+  contactLink,
+} from './modules/navigation.js';
+import { DateTime } from './modules/luxon.js';
+
+addBooks.style.display = 'none';
+contactUs.style.display = 'none';
+booksList.classList.add('active');
+booksList.classList.add('focus');
+
+listLink.addEventListener('click', () => {
+  window.location.reload();
+  addBooks.style.display = 'none';
+  contactUs.style.display = 'none';
+  booksList.style.display = 'flex';
+  booksList.classList.add('active');
+  booksList.classList.add('focus');
+});
+
+addNewLink.addEventListener('click', () => {
+  booksList.style.display = 'none';
+  contactUs.style.display = 'none';
+  addBooks.style.display = 'flex';
+  addBooks.classList.add('active');
+  addBooks.classList.add('focus');
+});
+
+contactLink.addEventListener('click', () => {
+  booksList.style.display = 'none';
+  contactUs.style.display = 'flex';
+  addBooks.style.display = 'none';
+  contactUs.classList.add('active');
+  contactUs.classList.add('focus');
+});
```

Comment:
[OPTIONAL] Nice work 💯 but this could be improved with a helper function. Here's a helper function that takes in the link element, the section element to display, and an array of classes to add to both the section element and the link element:
```
function displaySection(link, section, classes) {
  link.addEventListener('click', () => {
    // hide all sections
    booksList.style.display = 'none';
    contactUs.style.display = 'none';
    addBooks.style.display = 'none';

    // show selected section
    section.style.display = 'flex';

    // remove active and focus classes from all links
    listLink.classList.remove('active', 'focus');
    addNewLink.classList.remove('active', 'focus');
    contactLink.classList.remove('active', 'focus');

    // add active and focus classes to selected link
    link.classList.add('active', 'focus');

    // add classes to selected section
    classes.forEach(className => {
      section.classList.add(className);
      link.classList.add(className);
    });
  });
}
```
You can call this function for each link element and section element:
```
displaySection(listLink, booksList, ['active', 'focus']);
displaySection(addNewLink, addBooks, ['active', 'focus']);
displaySection(contactLink, contactUs, ['active', 'focus']);
```
This will simplify your code and make it more reusable if you need to add more link elements and sections in the future.